hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.test.tsx

```diff
@@ -86,11 +149,18 @@ describe("<ProductOverview />", () => {
         expect(getByText(minConstraint)).toBeInTheDocument();
     });
 
-    it("displays error message when unable to get hardware", () => {
-        const store = makeStore({
+    it("displays error message when unable to get hardware", async () => {
+        const apiResponse = makeMockApiResponse({}, 500);
```

Comment:
Could you remove this and the line below and replace it with this:

```
const failureResponse = {
            response: {
                status: 500,
                message: "Something went wrong",
            },
        };

        mockedGet.mockRejectedValue(failureResponse);
```

The problem is with how jest handles errors, right now ur mocking a "resolved value" meaning the get function was successful. Instead we need to mock a rejected value, that will cause the thunk to enter the catch block and throw handle an error